code/controllers/subsystem/ticker.dm

```diff
@@ -164,7 +175,7 @@ SUBSYSTEM_DEF(ticker)
 
 	if(hide_mode)
 		var/list/modes = new
-		for(var/datum/game_mode/M in runnable_modes)
+		for(var/datum/game_mode/M in config.get_runnable_modes())
```

Comment:
Doing `get_runnable_modes()` is pretty expensive. It would be best to do this only once. Moving the for loop to before the `runnable_modes` list gets mutated will work. Or you can use a copied list either here or make the copied list the one that gets emptied on line 144 etc